README.md

```diff
@@ -1,3 +1,19 @@
 # Safety-app
 
-Intermediary repo for the AGL cluster demo safety app, until meta-elisa is restructured
+The ELISA enhanced AGL instrument cluster demo offers an interface using a named pipe to trigger safe state from within the safety signal source. Alternatively the communication between signal source and safety app can be corrupted. This app is used to test the implemented mechanisms. 
+
+## Signalsource-control-panel interface
+
+Alternatively to writing to the named pipe directly, an ncurses based control panel can be used as more convenient alternative. 
```

Comment:
@pahmann This might sound better if we write it as:
`An ncurses based control panel can be used as a convenient alternative to writing to the named pipe directly`

Reply:
@sudipm-mukherjee thanks for the feedback. I will update the PR.